client.go

```diff
@@ -213,13 +214,26 @@ func (c *Client) TextToSpeech(voiceID string, ttsReq TextToSpeechRequest, querie
 		return nil, err
 	}
 	b := bytes.Buffer{}
-	err = c.doRequest(c.ctx, &b, http.MethodPost, fmt.Sprintf("%s/text-to-speech/%s", c.baseURL, voiceID), bytes.NewBuffer(reqBody), contentTypeJSON, queries...)
+	_, err = c.doRequest(c.ctx, &b, http.MethodPost, fmt.Sprintf("%s/text-to-speech/%s", c.baseURL, voiceID), bytes.NewBuffer(reqBody), contentTypeJSON, queries...)
 	if err != nil {
 		return nil, err
 	}
 	return b.Bytes(), nil
 }
 
+func (c *Client) TextToSpeechWithRequestID(voiceID string, ttsReq TextToSpeechRequest, queries ...QueryFunc) ([]byte, string, error) {
```

Comment:
Note, this new function was added to not break existing function's BC. If breaking BC is still acceptable, we can move this into the original function.